packages/backend/src/scheduler/SchedulerClient.ts

```diff
@@ -99,61 +93,66 @@ export class SchedulerClient {
 
         await graphileClient.completeJobs(deletedJobIds);
 
-        deletedJobs.rows.forEach(
-            ({
-                id,
-                scheduler_slack_target_uuid,
-                scheduler_email_target_uuid,
-                format,
-            }) => {
-                analytics.track({
-                    event: 'scheduler_job.deleted',
-                    anonymousId: LightdashAnalytics.anonymousId,
-                    properties: {
-                        jobId: id,
-                        schedulerId: schedulerUuid,
-                        schedulerTargetId:
-                            scheduler_slack_target_uuid ||
-                            scheduler_email_target_uuid ||
-                            'unknown',
-                        format: format || 'image',
-                        type:
-                            scheduler_slack_target_uuid !== null
-                                ? 'slack'
-                                : 'email',
-                    },
-                });
+        deletedJobs.rows.forEach(({ id }) => {
+            analytics.track({
+                event: 'scheduler_job.deleted',
+                anonymousId: LightdashAnalytics.anonymousId,
+                properties: {
+                    jobId: id,
+                    schedulerId: schedulerUuid,
+                },
+            });
+        });
+    }
+
+    private async addScheduledDeliveryJob(
+        date: Date,
+        scheduler: Scheduler,
+    ): Promise<void> {
+        const graphileClient = await this.graphileUtils;
+
+        const notification: ScheduledDeliveryPayload = {
```

Comment:
Is this variable name confusing? It's actually the the payload for the job that creates the csv/image rather than a single notification?